cmake/onnxruntime_framework.cmake

```diff
@@ -13,7 +13,7 @@ add_library(onnxruntime_framework ${onnxruntime_framework_srcs})
 if(onnxruntime_ENABLE_INSTRUMENT)
   target_compile_definitions(onnxruntime_framework PRIVATE ONNXRUNTIME_ENABLE_INSTRUMENT)
 endif()
-target_include_directories(onnxruntime_framework PRIVATE ${ONNXRUNTIME_ROOT} PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(onnxruntime_framework PRIVATE ${ONNXRUNTIME_ROOT} ${onnxruntime_CUDNN_HOME}/include PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
>${onnxruntime_CUDNN_HOME}/include [](start = 77, length = 33)

We don't want framework include provider, in this case CUDA related things. If needed, should we make a separate library that would be called CUDA framework?

Reply:
Yeah, right now cuda is used through the framework code. It's a future work item to move the cuda stuff into only the shared library.